### PR TITLE
fixing permanent NO-Internet issue

### PR DESCRIPTION
**Pull request by chebatory**

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -60,7 +60,7 @@ frontend:
 #
 network:
   in: iface_in
-  internet_check: https://1.1.1.1
+  internet_check: https://kaspersky.ru
```
Comment (felixaime) [inline config.yaml:63]: Why changing it to kaspersky.ru there? It can be filtered in some countries / networks.

**Comment by felixaime** [comment]
Hi, your installation breaks Raspbian Os (I don't know why, but tried several times and then my OS failed to boot). I'll try to make a better install file with conditions related to each OS.

File: config.yaml (hunk 1 above)
Comment (chebatory) [reply config.yaml]: Unfortunately 1.1.1.1 is blocked in Russia, however I agree with you that I should use some independent host, let use 8.8.8.8 then.

**Comment by chebatory** [comment]
> Hi, your installation breaks Raspbian Os (I don't know why, but tried several times and then my OS failed to boot). I'll try to make a better install file with conditions related to each OS.

Well I tried 10 times (on Raspbian, Debian, Ubuntu) and everything went fine. I noticed that my Raspberry sometimes failed to boot, however I`m sure that TinyCheck installation can not break whole system.

**Review by bzvr — APPROVED** [review]

**Review by bzvr — APPROVED** [review]

**Review by bzvr — APPROVED** [review]

**Pull request merged**